lib/livebook/runtime/evaluator.ex

```diff
@@ -689,25 +695,169 @@ defmodule Livebook.Runtime.Evaluator do
   end
 
   defp eval(:erlang, code, binding, env) do
+    case :erl_scan.string(String.to_charlist(code), {1, 1}, [:text]) do
+      {:ok, tokens, _} ->
+        case find_first_module_attribute(tokens) do
+          {:ok, _module_name} ->
+            eval_module(:erlang, code, tokens, binding, env)
+
+          :error ->
+            eval_statements(:erlang, code, tokens, binding, env)
```

Comment:
Both of these functions are erlang specific, so instead of an argument, we can make it clear in the function name :)

```suggestion
            eval_erlang_module(code, tokens, binding, env)

          :error ->
            eval_erlang_statements(code, tokens, binding, env)
```